### PR TITLE
Dev to main

### PR DESCRIPTION
**Pull request by jellywiz**
### In this project me @jellywiz and my partner @emmanuellmhango have implemented the following:

- Create React App.
- Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
- Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- Create routes and view components (Rockets, Missions, My Profile, Dragons _[only if your team has 3 members]_). Use `<NavLink />` for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).
- Create directories for all Redux state slice files (rockets, missions).
- Fetch data and update Redux store
- Once the data are fetched, dispatch an action to store the selected data in Redux store
- Render UI:lists
- Write actions and reducers for booking rockets and joining missions
- Write actions and reducers for canceling rocketsand leaving missions
- Render UI: conditional components rendering
- Render UI: My Profile section
- Technical set up

Link to the [Live Version](https://jellywiz.github.io/space-travlers/)
Link to the [Kanban](https://github.com/users/jellywiz/projects/6)

### CODE REVIEW

File: src/Components/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+.Nav-Item {
+  font-size: 1.3rem;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+.Nav-Item:active {
+  color: black;
+}
```
Comment (franclobo) [inline src/Components/Navbar.css]: - [ ] It is ok that you used a different color to indicate the active page, but please 🙏🏽 for a better user interface and to be similar to the original design use the underline to do it.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @jellywiz  and @emmanuellmhango, 

Good job so far!
You did it well 👏🏽 there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✅ 
- Descriptive PR 💯 
- Professional README file ☑️ 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/Components/Navbar.css (hunk 1 above)
Comment (jellywiz) [reply src/Components/Navbar.css]: Thanks for your feedback much appreciated ! @franclobo

**Review by Reem-lab — APPROVED** [review]
# Hi @jellywiz and @franclobo, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- No Linters errors ✅ 
- Followed Gitflow ✅
- Great working on the functionality and design🔥🚀🎉
- Professional README ✅
- Descriptive PR ✅
- Good working on the deployement ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jellywiz** [comment]
Thank you for your approval 😊 @Reem-lab

**Pull request merged**